code/datums/gamemodes/sovl_war.dm

```diff
@@ -0,0 +1,137 @@
+/datum/game_mode/infestation/sovl_war
+	name = "Sovl War"
+	config_tag = "Sovl War"
+	silo_scaling = 2
+	round_type_flags = MODE_ALAMO_ONLY|MODE_INFESTATION|MODE_LATE_OPENING_SHUTTER_TIMER|MODE_XENO_RULER|MODE_PSY_POINTS|MODE_PSY_POINTS_ADVANCED|MODE_HIJACK_POSSIBLE|MODE_SILO_RESPAWN|MODE_SILOS_SPAWN_MINIONS|MODE_ALLOW_XENO_QUICKBUILD|MODE_FORCE_CUSTOMSQUAD_UI
+	xeno_abilities_flags = ABILITY_NUCLEARWAR
+	valid_job_types = list(
+		/datum/job/terragov/command/captain = 1,
+		/datum/job/terragov/command/fieldcommander = 1,
+		/datum/job/terragov/command/staffofficer = 4,
+		/datum/job/terragov/command/pilot = 1,
+		/datum/job/terragov/command/transportofficer = 1,
+		/datum/job/terragov/engineering/chief = 1,
+		/datum/job/terragov/engineering/tech = 2,
+		/datum/job/terragov/requisitions/officer = 1,
+		/datum/job/terragov/medical/professor = 1,
+		/datum/job/terragov/medical/medicalofficer = 4,
+		/datum/job/terragov/medical/researcher = 2,
+		/datum/job/terragov/civilian/liaison = 2,
+		/datum/job/terragov/silicon/synthetic = 1,
+		/datum/job/terragov/command/transport_crewman = 1,
+		/datum/job/terragov/command/assault_crewman = 0,
+		/datum/job/terragov/silicon/ai = 1,
+		/datum/job/terragov/squad/engineer = 1,
```

Comment:
You should increase the number of engineers because theres more work for them with the FOB